src/components/legend/defaults.js

```diff
@@ -57,6 +57,7 @@ module.exports = function legendDefaults(layoutIn, layoutOut, fullData) {
     coerce('bordercolor');
     coerce('borderwidth');
     Lib.coerceFont(coerce, 'font', layoutOut.font);
+    coerce('orientation');
```

Comment:
I'd think bottom left but if we do this, we will need to take into account the case where range sliders are present. The legend is currently rendered below it so it gets hidden if it is placed in the same place.